CourseChamp-Frontend/src/components/CreateReview.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+    <div class="create-review-container">
+      <header>
+        <h1>Create A Review</h1>
+        <router-link to="/reviews" class="back-link">Back To Review</router-link>
+      </header>
+      <div class="form-container">
+        <form @submit.prevent="submitReview">
+          <div class="form-group">
+            <label class="input-label">Enter your rating:</label>
+            <input type="text" id="rating" v-model="rating" required placeholder="Ex: 5">
+          </div>
+          <div class="form-group">
+            <label class="input-label">Enter your review:</label>
+            <textarea id="text" v-model="text" required placeholder="Ex: I loved this class. It's a must take in U1! "></textarea>
+          </div>
+          <div class="msg">{{ msg }}</div>
+          <div class="submit-container">
+            <button type="submit" :disabled="!rating || !text" class="submit-button">
+              Submit
+              <span class="arrow-icon">
+                <img src="../assets/arrow.png" alt="Arrow">
+              </span>
+            </button>
```

Comment:
There's nowhere to specify which courseOffering this review is for right?

Reply:
Yeah exactly. I supposed the create review button would already be on the page of the course. However the features related to viewing reviews haven't been implemented yet.